pkg/upgrader/upgrade/kubernetes_binaries.go

```diff
@@ -59,8 +59,12 @@ curl -L "https://github.com/containernetworking/plugins/releases/download/{{ .CN
 
 RELEASE="v{{ .KUBERNETES_VERSION }}"
 
+sudo systemctl stop kubelet
```

Comment:
I'd prefer stopping kubelet only after download already finished and we know for real it's OK disk

Reply:
Done :)